packages/templates/src/cli.js

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-await-in-loop, no-console */
+
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import puppeteer from 'puppeteer';
+
+const screenshotsPath = `build/template-posters/`;
+
+fs.mkdirSync(screenshotsPath, { recursive: true });
+
+(async () => {
+  const browser = await puppeteer.launch({
+    defaultViewport: null,
+    headless: true,
+    args: [`--window-size=1600,800`],
+  });
+  const page = await browser.newPage();
+  await page.setViewport({
+    width: 1600,
+    height: 800,
+    deviceScaleFactor: 2,
+  });
+  await page.goto(
+    'http://localhost:8899/wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/templates-gallery'
+  );
```

Comment:
Can't we use [e2e-test-utils](https://github.com/google/web-stories-wp/blob/main/packages/e2e-test-utils) here?

Like [visitDashboard](https://github.com/google/web-stories-wp/blob/main/packages/e2e-test-utils/src/visitDashboard.js)